site/access-control.xml

```diff
@@ -328,7 +391,17 @@ limitations under the License.
             href="https://github.com/gotthardp/rabbitmq-auth-backend-ip-range">source
             IP range backend</a> for authorisation:
           </p>
-            
+
+<pre class="example">
+# rabbitmq.conf
+#
+auth_backends.1.authn = internal
+# uses module name because this backend is from a 3rd party
+auth_backends.1.authz = rabbit_auth_backend_ip_range
```

Comment:
I don't know enough how authentication and authorization are handled by RabbitMQ but this configuration parameters look reversed.

I would expect something like:
```
auth_backends.authn.1 = internal
auth_backends.authn.2 = http
auth_backends.authz.1 = rabbit_auth_backend_ip_range
```
In other words: RabbitMQ checks authentication first using the listed modules, then it checks authorization using another set of modules. Of course, correct me if my understanding of RabbitMQ internals is wrong.